generator/cli/generate-migration-scripts.ts

```diff
@@ -1,6 +1,8 @@
+import { FileTransformers } from '../../src/fileTransformers'
 import { Reflector } from '@prisma-spectrum/reflector'
 import { PromisePool } from '@supercharge/promise-pool'
 import { PrismaTemplates } from '~/src'
+import { FileTransformer } from '~/src/fileTransformer'
```

Comment:
Importing this will not work because it creates a 🐔 🥚 situation. `~/src/utils` works because that module graph does not reach into any generated code.

I have some ideas about how to restructure the codebase to make these separations clearer but out of scope for now.